frontend/src/scenes/trends/personsModalLogic.tsx

```diff
@@ -129,6 +129,7 @@ export const personsModalLogic = kea<personsModalLogicType<LoadPeopleFromUrlProp
         setSearchTerm: (term: string) => ({ term }),
         setCohortModalVisible: (visible: boolean) => ({ visible }),
         loadPeople: (peopleParams: PersonsModalParams) => ({ peopleParams }),
+        setUrl: (props: LoadPeopleFromUrlProps) => props,
```

Comment:
always objects for actions plis!
```suggestion
        setUrl: (props: LoadPeopleFromUrlProps) => ({props}),
```

Reply:
Huh, wait, this is weird, in the reducer below, it passes the props, not these props^. What?

Reply:
done